content/docs/reference/apis/_index.en.md

```diff
@@ -0,0 +1,13 @@
+---
+title: "APIs"
+linkTitle: "APIs"
+description: "Programming interfaces specifications"
+weight: 60
+---
+
+RailJSON is the format used to describe a railway infrastructure, it's
+described in its [JSON schema].
+
+Below are a list of REST APIs implemented by OSRD.
+
+[JSON schema]: https://json-schema.app/view/%23?url=https%3A%2F%2Fraw.githubusercontent.com%2FOpenRailAssociation%2Fosrd%2Fdev%2Ffront%2Fsrc%2Freducers%2Fosrdconf%2Finfra_schema.json
```

Comment:
It would maybe be possible to use a HTML template instead of lazily linking to the deployed webapp, let me know if that would be preferred.

Reply:
This is more than enough.